README.es-ES.md

```diff
@@ -0,0 +1,571 @@
+# Angular Performance Checklist
+
+<img src="./assets/flash.png" width="1000">
+
+- [中文版](./README.zh-CN.md)
+- [Русский](./README.ru-RU.md)
+- [Português](./readme-pt-BR.md)
+- [Español](./readme-es-ES.md)
+
+## Introducción
+
+Este documento contiene una lista de prácticas las cuales nos ayudarán a aumentar el rendimiento de nuestras aplicaciones Angular. "Angular performance Checklist" cubre diferentes temas - desde server-side pre-rendering y bundle de nuestras aplicaciones, hasta redimiento en ejecución y optimización de la detección del cambio realizada por el framework.
+
+El documento se divide en dos secciones principales:
+
+- Rendimiento de la red - lista de prácticas que mejorarán principalmente el tiempo de carga de nuestra aplicación. Incluyen métodos de latencia y reducción de ancho de banda.
+- Rendimiento en ejecución - Prácticas que mejoran el rendimiento en ejecución de nuestra aplicación. Incluyen principalmente optimizaciones en la detección del cambio y relacionadas con el renderizado.
+
+Algunas prácticas impactan en ambas categorías por lo que podría haber un ligero punto de encuentro, sin embargo, la diferencia en los casos de uso y las implicaciones serán explícitamente mencionadas.
+
+La mayoría de subsecciones enumeran herramientas, relacionadas a la práctica específica, que nos harán más eficiente la automatización de nuestro entorno de desarrollo.
+
+Tenga en cuenta que la mayoría de prácticas son válidas para HTTP/1.1 y HTTP/2. Se mencionarán las prácticas que sean la excepción especificando a qué versión del protocolo podrían aplicarse.
```

Comment:
```suggestion
Tenga en cuenta que la mayoría de prácticas son válidas para HTTP/1.1 y HTTP/2. Se mencionarán las prácticas que sean la excepción especificando a que versión del protocolo podrían aplicarse.
```

Reply:
Tilde is being correctly used in the original.

Reply:
Me parece que en este caso ambas son válidas.

Reply:
Si se puede reemplazar por "cuál" entonces se escribe "qué"

http://www.rae.es/consultas/tilde-en-que-cuales-quienes-como-cuan-cuantoaosas-cuando-donde-y-adonde